openjdk/playlist.xml

```diff
@@ -2221,6 +2221,14 @@
 	</test>
 	<test>
 		<testCaseName>jdk_vector</testCaseName>
+		<disables>
+			<disable>
+				<comment>https://github.com/eclipse-openj9/openj9/issues/19360</comment>
+				<impl>openj9</impl>
+				<impl>ibm</impl>
+				<version>17</version>
+			</disable>
+		</disables>
```

Comment:
Unfortunately, we cannot have 2 impl in one `<disable>`. They have to be separated.
```
<disables>
	<disable>
		<comment>https://github.com/eclipse-openj9/openj9/issues/19360</comment>
		<impl>openj9</impl>
		<version>17</version>
	</disable>
	<disable>
		<comment>https://github.com/eclipse-openj9/openj9/issues/19360</comment>
		<impl>ibm</impl>
		<version>17</version>
	</disable>
</disables>
```

Reply:
Updated